pom.xml

```diff
@@ -14,7 +14,7 @@
     <name>Andersen</name>
     <description>Andersen</description>
     <properties>
-        <java.version>17</java.version>
+        <java.version>1.8</java.version>
     </properties>
```

Comment:
Any reason for decrease java version?

Reply:
no objective reasons, it was just one of my attempts to make something that is no important anymore